<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntry_Advances</class>
 <widget class="QMainWindow" name="DataEntry_Advances">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advances</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/1522_stop_watch_48x48.png</normaloff>:/images/1522_stop_watch_48x48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="cmdPrevious">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPos">
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cmdNext">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdNew">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="txtMasterDetails">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>151</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Advance List</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="cmdNewAdvance">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../Leva/src/res.qrc">
            <normaloff>:/images/Icon_193.ico</normaloff>:/images/Icon_193.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="cmdDeleteAdvance">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../Leva/src/res.qrc">
            <normaloff>:/images/Icon_194.ico</normaloff>:/images/Icon_194.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tblAdvances"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>label_3</zorder>
   <zorder>tblAdvances</zorder>
   <zorder>txtMasterDetails</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../Leva/src/res.qrc"/>
  <include location="res.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
